pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1144,6 +1154,13 @@ type Service struct {
 	Type string `json:"type,omitempty"`
 }
 
+// SuspendAction defines the suspend actions for a component.
+//
+// +k8s:openapi-gen=true
+type SuspendAction struct {
+	SuspendStatefuleSet bool `json:"suspendStatefulset,omitempty"`
```

Comment:
```suggestion
	SuspendStatefulSet bool `json:"suspendStatefulset,omitempty"`
```

Reply:
Done, and update `suspendStatefulset` to `suspendStatefulSet`